src/screens/SendCoinsScreen.tsx

```diff
@@ -168,10 +168,14 @@ class SendCoinsScreen extends Component<Props, State> {
     return (availableBalance === 'NaN' && balance) || availableBalance;
   }
 
+  toNumber = (text?: string) => Number(text?.replace(',', '.'));
+
   validateTransaction = (): string | null => {
     const { fee: requestedSatPerByte, transaction, wallet } = this.state;
 
-    if (!transaction.amount || transaction.amount <= 0) {
+    const numAmount = this.toNumber(transaction.amount);
+
+    if (!transaction.amount || Number.isNaN(numAmount) || numAmount <= 0) {
```

Comment:
That would be enough I think

`if (!numAmount || numAmount <= 0)`

`!NaN` -> true

Reply:
changed